report_qweb_field_option/__manifest__.py

```diff
@@ -1,13 +1,16 @@
-# Copyright 2024-2025 Quartile (https://www.quartile.com)
+# Copyright 2024-2025 Quartile (https://www.quartile.co)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
-    "name": "Report Qweb Field Options",
+    "name": "Report Qweb Field Option",
     "version": "16.0.1.0.0",
     "category": "Technical Settings",
     "license": "AGPL-3",
     "author": "Quartile, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/reporting-engine",
     "depends": ["uom"],
+    "external_dependencies": {
+        "python": ["odoo_test_helper"],
```

Comment:
@AungKoKoLin1997 I think we should remove this dependency from the manifest file, as it forces inconveniences of including the module in the repo. It should be OK as long as we have test_requirements.txt updated. What's your take on this?